src/query-compiler/default-query-compiler.ts

```diff
@@ -1428,7 +1428,11 @@ export class DefaultQueryCompiler
       this.append('if exists ')
     }
 
-    this.visitNode(node.name)
+    this.compileList([node.name, ...(node.additionalNames ?? [])])
```

Comment:
Let's simplify this a bit.

```suggestion
    this.visitNode(node.name)
    
    if (node.additionalNames?.length) {
      this.append(', ')
      this.compileList(node.additionalNames)
    }
```